vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -42,15 +42,31 @@
  *
  */
 public final class Serializer {
-
+	/**
+	 * Constructor.
+	 */
 	private Serializer() {
 
 	}
 
+	/**
+	 * Creates a String representation of a given {@link Rule}. Example: "p(?X) :-
+	 * q(?X,?Y)."
+	 * 
+	 * @param rule a {@link Rule}
+	 * @return String representation corresponding to a given {@link Rule}.
+	 */
 	public static String getString(Rule rule) {
 		return getString(rule.getHead()) + " :- " + getString(rule.getBody()) + ".";
 	}
 
+	/**
+	 * Creates a String representation of a given {@link Literal}. Example:
+	 * "~q(?X,?Y)"
```

Comment:
Please explain that this is an example of a NegativeLiteral, and show and example of a PositiveLiteral.

Also, include constants in the example.